Source/engine/assets.cpp

```diff
@@ -63,8 +63,8 @@ SDL_RWops *OpenAsset(const char *filename, bool threadsafe)
 	if ((rwops = SDL_RWFromFile(path.c_str(), "rb")) != nullptr)
 		return rwops;
 
-#ifdef __ANDROID__
-	// On Android, fall back to the APK's assets.
+#if defined(__ANDROID__) || defined(__APPLE__)
+	// On Android and iOS, fall back to the APK/Bundle's assets.

```

Comment:
This applies to all of Apples OSs (macOS, tvOS, watchOS and iOS), and hopefully we can also use this code path for other systems that has a build in package format.
```suggestion
	// On fall back to the bundled assets on supported systems.
```